### ISSUE TITLE
Feature request: alarm clock

### ISSUE THREAD

**Issue by sudomain** [orgzly/orgzly-android#654]
Most notification sounds are not sufficient to wake someone from sleep. Many devices have a stock alarm clock app which might be adequate, but having the scheduling abilities of Orgzly would be better. It would be great if we could utilize stock alarm tones for certain notes, as well as use notification sounds for other notes. As far as I can tell, there isn't an option to set different notification sounds on a per-note basis so maybe I need to create a separate issue for that?
- [ x] I have searched for existing issues that may be the same as or related to mine.

**Comment by ScottFreeCode** [comment]
Android even has an API for the alarms, so that, in a nutshell, you could schedule real/ordinary alarms from an additional app (such as Orgzly).

On the other hand I'm not sure what would be a good convention to use to indicate an alarm, and not merely a regular reminder, in Org format (considering the canonical "app" for the format is Org Mode on Emacs).

My personal workaround is to keep setting ordinary alarms for when I want to be woken up, even if I have Orgzly reminders scheduled to be shown at the time I should wake up (i.e. the same time). I'll admit that's redundant and as a programmer it's always tempting to automate redundancy away…

**Comment by Ypot** [comment]
I don't see "[scheduling](https://orgmode.org/manual/Deadlines-and-scheduling.html)" as an appropriate way of setting alarms. Maybe Timestamps could be more appropriate, but by now they don't work on orgzly.

**Comment by nevenz** [comment]
> On the other hand I'm not sure what would be a good convention to use to indicate an alarm, and not merely a regular reminder, in Org format (considering the canonical "app" for the format is Org Mode on Emacs).

Yeah, a custom property would work, though I've been trying to avoid those.

**Comment by vit1-irk** [comment]
@nevenz @ScottFreeCode 
You can use not custom property, but the tags mechanism, natively integrated both in Orgzly and org-mode. In the Orgzly settings screen may exist the property "tags which trigger an alarm", as already implemented with active/done states like TODO, NEXT, DONE. If org-mode entry triggers notification due to timestamp or scheduled/deadline thing, Orgzly can fire an alarm in the case one of these tags are presented.

------
Speaking generally about the issue, alarm clock feature is very important thing, because it makes you capable to control not only your appoinments/notes/schedule/plans using plaintext, but also sleeping habits. Org-mode supports scripting and even data science things which makes it so powerful. Just imagine you can adjust and automate every tiny detail of your life _using only one or few text files_.  Does it sound amazing enough? 😌

As I searched on F-Droid or any other FLOSS stores, I didn't find any sync-capable alarm apps. 90% of them are just offline and can't even export/import the settings. So, every f**king time you reinstall the ROM or migrate data between devices or remember that "next 2 weeks I have another schedule", you have to set up all the alarms from scratch.

It will be very convenient to set up alarms in Emacs on desktop (having all the stuff organized in 1 place), then save buffer (it gets synchronized with Android via Syncthing/WebDav) and instantly enjoy your _everything_ available on mobile device.

**Comment by doak** [comment]
Furthermore, a tag based selection would support to configure it differently for a second device.
(I sync notebooks with my family etc.)

**Comment by odinu** [comment]
According to the latest release notes [1.8.10](https://github.com/orgzly/orgzly-android/releases/tag/v1.8.10) this has been implemented:
```
Allow using alarm clock for scheduling reminders with a time of day
```
 Doesn't seem to be working for me though. Did anyone manage to get it working?

**Comment by kisp** [comment]
@dncrash I just tried, and for me, it also doesn't work.

**Comment by zaz** [comment]
Thank you for all your work on Orgzly!

The alarm clock feature was added in 9ecaee04d11d341bb924ee3d17ff583f8c504713, so could be relabel this issue as a fix rather than a feature request?
